CMakeLists.txt

```
@@ -38,7 +38,7 @@ if(NOT CMAKE_CXX_STANDARD)
 endif(NOT CMAKE_CXX_STANDARD)
 
 # https://github.com/izenecloud/cmake/blob/master/SetCompilerWarningAll.cmake
-if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+if(MSVC)
```

see https://cmake.org/cmake/help/v3.0/variable/CMAKE_CL_64.html
```suggestion
if(CMAKE_CL_64)
```

---

CMAKE_CL_64 is 64bit only